demo.html

```diff
@@ -5,12 +5,24 @@
 description: "Examples of Signal K web apps."
 ---
 
+<h2>Signal K Demonstrator</h2>
 <p>
-  <a href="http://demo.signalk.org" target="_blank">Open in new tab <i class="fa fa-external-link"></i></a>
+  Currently most of the "cutting edge" development is being done on the Signal K Node-Server. Node is a popular development
+  platform based on Google Chrome's JavaScript runtime environment and it can run on a lot of different hardware including
+  x64, arm, arm64, ppc64le, power7, power8, and s390x.</p>
+<p>
+  One of the most popular hardware platforms for Signal K is the Raspberry Pi, which on a "Processing Power to Dollar" ratio,
+  is pretty unbeatable and with so many people using it, getting help and support is much easier than for other platforms.</p>
+<p>
+  For those that prefer working in Java, there is also a Signal K Java-Server, but it is currently not fully up to date with the
+  latest development work.</p>
+<p>
+  A fully working version of the latest Signal K Node-Server is currently running on the Signal K demo webserver and if you click
+  on the link below it will open in a new tab. You will be taken to the Server's "Dashboard" where you can then click on the
+  Webapps menu to see some typical Signal K Apps.</p>
+<p>
+  This is the same Node Server that you could be running on a Raspberry Pi on your boat and it would be your boat's real data
+  being displayed in your web browser, rather than our demo data.</p>
+<p>
+<h3><a href="http://node-master.signalk.org" target="_blank">***Click here to open Signal K Node-Server in a new tab***<i class="fa fa-external-link"></i></a>
```

Comment:
Markdown doesn't work inside of HTML. If you want this to be bold, add a class to the link. Better would be to make it a button and remove the &lt;h3&gt;.

Also, this should link to the stable version of Node server (I think), so
```
<a href="https://demo.signalk.org" target="_blank" class="button-big" title="Open Signal K Node Server in a new tab">Open Signal K Node Server <i class="fa fa-external-link"></i></a>
```
and the title attribute will give you simple tooltip.